ush/config.sh.3DRTMA_dev1

```diff
@@ -75,21 +75,29 @@ SFCOBS_USELIST="/home/amb-verif/ruc_madis_surface/mesonet_uselists"
 #
 QUILTING="TRUE"
 CCPP_PHYS_SUITE="FV3_HRRR"
+
+EXTRN_MDL_ICS_OFFSET_HRS="0"
+BOUNDARY_LEN_HRS="6"
+BOUNDARY_LONG_LEN_HRS="6"
+LBC_SPEC_INTVL_HRS="1"
+EXTRN_MDL_LBCS_OFFSET_HRS="0"
+EXTRN_MDL_LBCS_SEARCH_OFFSET_HRS="0"
+
+#INITIAL_CYCLEDEF="00 00,12 * * 2021 *"
+#BOUNDARY_CYCLEDEF="00 00-02/01,04-08/01,10-14/01,16-20/01,22,23 * * 2021 *"
+#BOUNDARY_LONG_CYCLEDEF="00 03,09,15,21 * * 2021 *"
+#PREP_COLDSTART_CYCLEDEF="00 00,12 * * 2021 *"
```

Comment:
change
`#`PREP_COLDSTART_CYCLEDEF="00 00,12 * * 2021 *"
to
PREP_COLDSTART_CYCLEDEF="00 00,12 * * `2100` *"
if you will make another update.

Reply:
The default CYCLEDEF is "00 01 01 01 2100 *". You don't need to set this if this one is not used.
Also, please don't use * * in day and month unless you need. It tells rocoto to generate one year of prep_coldstart group. The default only generate one cycle.

Reply:
ok got it. Have to remember it has a default value.